Azure/azure-cli
# [RDBMS] `az mysql flexible-server create/update`: Add `--auto-scale-iops` to enable or disable autoscale of iops

**Related command**

We add a new parameter **auto-scale-io** for the server create/update command. This parameter is enum. 
- az mysql flexible-server create/update --auto-scale-io [Enabled/Disabled]

**Description**

This PR uses latest Python SDK package which in turn uses the latest GA API version 2022-12-01. This PR will allow customers to use the new feature: Auto scale iops when creating and updating servers.

**Testing Guide**

See examples in _helptext_mysql.py.



---

This checklist is used to make sure that common guidelines for a pull request are followed.
